stack.yaml

```diff
@@ -70,24 +70,32 @@ extra-deps:
 - repline-0.2.1.0
 - git: https://github.com/target/row-types
   commit: 1e8d5e084ffd46f6c7842826a1f62c60820885df
+  # nix-sha256: 0ly5m4r8wkm8gdqyrqzsjfmp189yxsd4qp0zi3idrbgfaf45sk9k
 # Needs some patches, but upstream seems to be fairly dead (no activity in > 1 year)
 - git: https://github.com/shmish111/purescript-bridge.git
   commit: 28c37771ef30b0d751960c061ef95627f05d290e
+  # nix-sha256: 0n6q7g2w1xafngd3dwbbmfxfn018fmq61db7mymplbrww8ld1cp3
 - git: https://github.com/shmish111/servant-purescript.git
   commit: ece5d1dad16a5731ac22040075615803796c7c21
+  # nix-sha256: 1axcbsaym64q67hvjc7b3izd48cgqwi734l7f7m22jpdc80li5f6
 - git: https://github.com/input-output-hk/cardano-crypto.git
-  commit: 3c5db489c71a4d70ee43f5f9b979fcde3c797f2a
+  commit: 2547ad1e80aeabca2899951601079408becbc92c
```

Comment:
Did you mean to bump this?

Reply:
Yes, this moves it past the openssl drop. See https://github.com/input-output-hk/cardano-crypto/pull/65